controllers/reviewsControllers.js

```diff
@@ -36,4 +36,19 @@ exports.getReviewsById = (request, response, next) => {
 		.catch((error) => {
 			next(error);
 		});
-};
\ No newline at end of file
+};
+
+exports.getCommentsByReview=(request,response,next)=>{
+	const {review_id}=request.params
+	const queryStringComments=format('SELECT * FROM comments ORDER BY created_at DESC')
+	fetchData(queryStringComments).then((comments)=>{
+		const newArr=[]
+		comments.forEach((comment)=>{
+			if(comment['review_id']===parseInt(review_id)){
```

Comment:
We do not want to be doing a for each on our data to find the ones with the relevant ID. We can do this within our SQL with a WHERE clause :)

Reply:
You will also want a new model for this controller